pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -351,14 +356,15 @@ In the program state
 
 evaluation cannot depend on @racket[<o2>], because it is not part of
 the program to evaluate, and it is not referenced by any definition
-that is accessible in the program. The @tech{object} @racket[<o2>] may
-therefore be removed from the evaluation by @deftech{garbage
+that is accessible by the program. The object is said to be
+not @deftech{reachable}. The @tech{object} @racket[<o2>] may
+therefore be removed from the program state by @deftech{garbage
```

Comment:
I like this `deftech` here, though I think it would be *slightly* less clumsy to write “said to not be reachable” rather than “said to be not reachable”. I wonder if a simple text search through the reference could find places to add `@tech{reachable}` where appropriate?

Reply:
I didn't use `@tech` around all the instances in the next few paragraphs, since the reader's attention is already on the term. I did add it around an instance in the "Custodians" section.